the_well/utils/download.py

```diff
@@ -2,14 +2,16 @@
 import glob
 import os
 import subprocess
-from typing import Optional
+from typing import List, Optional
 
 import yaml
 
 from the_well.data.utils import WELL_DATASETS
 
 WELL_REGISTRY: str = os.path.join(os.path.dirname(__file__), "registry.yaml")
 
+STAT_FILE_KEY = "stats"
```

Comment:
I am not sure this global variable is very useful. If you think it is actually necessary to modify that key, an argument in the functions (`create_url_registry` and `well_download`) would be more useful.

Reply:
Agreed. I thought it would avoid duplicates of `stats` entry but may not be that useful.